app.js

```diff
@@ -180,6 +180,30 @@ function get_widgets(SETTINGS) {
         sortBy: function(a, b){ return PLAYING_TIME_ORDER.indexOf(a.name) - PLAYING_TIME_ORDER.indexOf(b.name); },
       }
     ),
+    "refine_previousplayers": panel('Previous players')(instantsearch.widgets.refinementList)(
+      {
+        container: '#facet-previous-players',
+        attribute: 'previous_players',
+        operator: 'and',
+        searchable: true,
+        showMore: true,
+      }
+    ),
+    "refine_numplays": panel('Total plays')(instantsearch.widgets.numericMenu)(
+      {
+        container: '#facet-numplays',
+        attribute: 'numplays',
+        items: [
+          { label: 'Any' }, 
+          { label: '0', end: 0 },
+          { label: '1', start: 1, end: 1 },
+          { label: '2-9', start: 2, end: 9 },
+          { label: '10-19', start: 10, end: 19 },
+          { label: '20-39', start: 20, end: 29 },
```

Comment:
Typo: should be 29, not 39.

Reply:
Oops, good catch. I've gotta get better at proof-reading before hitting commit!

Reply:
That's why we have code review! :)